src/components/EventList.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import { mount } from '@cypress/react';
+
+import EventList from './EventList';
+
+describe('EventList', () => {
+    let dummyOrg;
+    let dummyEvents;
+
+    beforeEach(()=> {
+        cy.fixture('dummyOrg.json')
+            .then((data) => {
+                dummyOrg = data;
+            });
+        cy.fixture('dummyEvents.json')
+            .then((data) => {
+                dummyEvents = data;
+            });
+    });
+
+    it('contains data for each event', () => {
+        mount(<EventList events={ dummyEvents.data } org={ dummyOrg.data }/>);
+
+        cy.get('[data-test="event-list"]>[data-test="event"]').each((item) => {
+            cy
+                .wrap(item)
```

Comment:
Personally I'd prefer these to be written on one line, i.e. `cy.wrap(item)`. I think `cy` and `.wrap(item)` feel so disconnected when written like this, which makes it more difficult to read IMHO.

Reply:
Yes, looks better.